README.md

```diff
@@ -98,7 +100,58 @@ https://community.mattermost.com/plugins/jira/webhook?secret=5JlVk56KPxX629ujeU3
 
 ### 3.2 Jira Server (Core + Software)
 
-Documentation to be added.
+#### 3.2.1 Mattermost
+
+1. Download this binary: https://s3.amazonaws.com/mattermost-public-plugins-kubernetes/jira-test-2.0.0.tar.gz
+2. Go to **System Console > Plugins (Beta) > Management** and upload the plugin from step 1. If you don't have the ability to upload plugins, uploads may be disabled on your server. To enable them, set **PluginSettings > EnableUploads** to `true` in your `config.json` file.
+3. Once uploaded, the Jira plugin will appear in a list of installed plugins. Click **Enable** to enable it.
+4. Go to **System Console > Plugins (Beta) > Management > Jira**, select the username that this plugin is attached to, and enter your Jira Server URL.
```

Comment:
Step 4. assumes users have already created the username to attach to the plugin.  Should we add instructions for creating the username?